specification/postgresql/resource-manager/readme.python.md

```diff
@@ -10,12 +10,12 @@ no-namespace-folders: true
 clear-output-folder: true
 ```
 
-### Tag: package-flexibleserver-2024-11-01-preview and python
+### Tag: package-flexibleserver-2024-08-01 and python
```

Comment:
>2024-08-01 [](http://example.com/codeflow?start=32&length=10)

curious: is the API version being used is going back to older api-version.

Reply:
![image](https://github.com/user-attachments/assets/feee8917-4f2a-46e3-a3a7-55fc007e4f6e)
SDK team receives release request from service team to release this stable tag `package-flexibleserver-2024-08-01`.